src/main/hbase_region_inspector/hbase/base.clj

```diff
@@ -1,29 +1,33 @@
-(ns hbase-region-inspector.hbase.base)
+(ns hbase-region-inspector.hbase.base
+  (:import [org.apache.hadoop.hbase
+            RegionLoad]
+           [org.apache.hadoop.hbase
+            HRegionInfo]))
 
 (defn info->map
   "Builds map from HRegionInfo"
   [info]
-  {:encoded-name (.getEncodedName info)
+  {:encoded-name (.getEncodedName ^HRegionInfo info)
    ; :table
-   :start-key    (.getStartKey info)
-   :end-key      (.getEndKey info)
-   :meta?        (.isMetaRegion info)})
+   :start-key    (.getStartKey ^HRegionInfo info)
+   :end-key      (.getEndKey ^HRegionInfo info)
+   :meta?        (.isMetaRegion ^HRegionInfo info)})
 
 (defn load->map
   "Builds map from RegionLoad"
   [load]
-  {:compacted-kvs              (.getCurrentCompactedKVs load)
-   :memstore-size-mb           (.getMemStoreSizeMB load)
-   :read-requests              (.getReadRequestsCount load)
-   :requests                   (.getRequestsCount load)
-   :root-index-size-kb         (.getRootIndexSizeKB load)
-   :store-file-index-size-mb   (.getStorefileIndexSizeMB load)
-   :store-files                (.getStorefiles load)
-   :store-file-size-mb         (.getStorefileSizeMB load)
-   :stores                     (.getStores load)
+  {:compacted-kvs              (.getCurrentCompactedKVs ^RegionLoad load)
```

Comment:
Similarly, `[^RegionLoad load]` is preferable.

Reply:
done